src/client/index.ts

```diff
@@ -75,6 +107,15 @@ export class DoubleUpClient {
     getLimboResult = getLimboResult;
 
     // plinko
-    createPlinko = (input: PlinkoInput) => createPlinko({ ...input, plinkoPackageId: this.plinkoPackageId });
-    getPlinkoResult = getPlinkoResult;
+    createPlinko = (input: PlinkoInput) => createPlinko({
+      ...input,
+      plinkoPackageId: this.plinkoPackageId,
+      plinkoVerifierId: this.plinkoVerifierId
+    });
+    getPlinkoResult = (input: PlinkoResultInput) => getPlinkoResult({
+      ...input,
```

Comment:
where does this 

```
...input
```

come from?

Reply:
the sdk game functions require variables (e.g. sui client, package IDs) that i didn't want people to have to keep track of over and over.

the class holds onto those values and uses them directly on behalf of the user.

anyone using this sdk will call something like this:

```
const input = {
  coinType: "...",
  gameSeed: "...",
  transactionResult
};

await getPlinkoResult(input);
```

but inside the sdk it will call it like this:

```
await getPlinkoResult({
  coinType: "...",
  gameSeed: "...",
  transactionResult,

  plinkoPackageId: "...",
  plinkoCorePackageId: "...",
  suiClient
});
```

in this case `...input` is what another developer uses to interface with the sdk, like the first example above.

Reply:
```
getPlinkoResult = (input: PlinkoResultInput) => getPlinkoResult({ ... });
                                ^^^^^
```